### ISSUE TITLE
pacman.rbf

### ISSUE THREAD

**Issue by NML32** [Gehstock/Mister#1]
Pacman has a humming sound that appears to be louder when pacman is eating pellets. 
Playing sound from HDMI with default settings. 
As far as I can tell everything else seems fine.

**Comment by NML32** [comment]
I hope you find time to port more of your MiST cores to MiSTer.

**Comment by Gehstock** [comment]
Yes hope Problem is fixed now(in Source), but i cant build new Cores at this Time need a new Computer.

**Comment by Gehstock** [comment]
Is the VGA Output for this Cores ok? i Can´t test.

**Comment by NML32** [comment]
I think it is, I’ll check again when I get home.

**Comment by Gehstock** [comment]
ok thanks
